### PR TITLE
Fix IndexOutOfBoundsException issue #1584

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt [hunk 1]
```diff
@@ -36,15 +35,15 @@ class EllipsizedTextView @JvmOverloads constructor(
...
-        var ellipsizedText = TextUtils.ellipsize(text, paint, textWidth, ellipsize)
+        var ellipsizedText = TextUtils.ellipsize(text, paint, screenWidth, ellipsize)
 
         if (ellipsizedText != text) {
-            textWidth = screenWidth
-            ellipsizedText = TextUtils.ellipsize(text, paint, textWidth, ellipsize)
-
-            builder.clear()
-            text = builder.append(ellipsizedText).replace(0, 1, "")
+            ellipsizedText = TextUtils.ellipsize(text, paint, screenWidth, ellipsize)
```
Comment (Yuseon-Han) [inline nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt:41]: why this line be called executed again?

Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt:41]: Right, it's unnecessary code

Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt]: Apply review~ thank you!

Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt]: > why this line be executed again?

Done! thank you!

**Review by Yuseon-Han — APPROVED** [review]

**Review by shshshl09 — APPROVED** [review]

**Pull request merged**